packages/guile.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Guile < Package
+  description 'Guile is designed to help programmers create flexible applications that can be extended by users or other programmers with plug-ins, modules, or scripts.'
+  homepage 'http://www.gnu.org/software/guile/'
+  version '2.2.3'
+  source_url 'https://ftp.gnu.org/gnu/guile/guile-2.2.3.tar.xz'
+  source_sha256 '8353a8849cd7aa77be66af04bd6bf7a6207440d2f8722e46672232bb9f0a4086'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'libtool' => :build
+  depends_on 'bdwgc'
+  depends_on 'libffi'
+  depends_on 'libunistring'
```

Comment:
Most lisps need readline or rlwrap for their REPL to be useful.

Reply:
Those weren't explicitly required in the configure script.  Can you please give an example of something that doesn't work using one of the guile executables?